go/vt/vtgate/vindexes/vschema_test.go

```diff
@@ -2637,7 +2690,12 @@ func TestVSchemaPBJSON(t *testing.T) {
 		t.Error(err)
 	}
 	want := vschemapb.Keyspace{
-		Sharded: true,
+		Sharded:        true,
+		ForeignKeyMode: vschemapb.Keyspace_unmanaged,
+		MultiTenantSpec: &vschemapb.MultiTenantSpec{
+			TenantIdColumnName: "tenant_id",
+			TenantIdColumnType: "int64",
```

Comment:
Will let @deepthi decide what we want to do here. Not sure if we expose types for other functionality today.

Reply:
Good catch.
Instead of column_type being a string, you can use query.Type. That is what we use for table columns in other places in the vschema proto.

Reply:
So the vschema would need to be defined as
```json
  "multi_tenant_spec": {
      "tenant_id_column_name": "tenant_id",
      "tenant_id_column_type": 265
  },
  "tables": {
    "t1": {}
  }
```

Is that ok for enduser UX? Or is there some other mechanism for them to define the type using the literal instead of using the enum value?